cross/openssl/Makefile

```diff
@@ -2,7 +2,7 @@ PKG_NAME = openssl
 PKG_VERS = 1.0.2n
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://www.openssl.org/source
+PKG_DIST_SITE = https://www.openssl.org/source/old/1.0.2/
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

Comment:
Just a suggestion:
```
OLD_VERSION = 1.0.2
PATCH_VERS = n
PKG_VERS = $(OLD_VERSION)$(PATCH_VERS)

PKG_DIST_SITE = https://www.openssl.org/source/old/$(OLD_VERSION)
```

Reply:
Thanks for the comment. I've applied your suggestion in a separate commit but I haven't done any build yet to test. Will do later if I have the time :)